.github/workflows/generate_missing_mappings.yml

```diff
@@ -0,0 +1,118 @@
+name: Generate and commit taxonomy mapping changes to requested PR
+
+on:
+  issue_comment:
+    types: [created]
+
+concurrency:
+  group: "generate_mappings"
+  cancel-in-progress: false
+
+jobs:
+  generate_missing_mappings:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: read
+    if: ${{github.event.issue.pull_request && github.event.comment.body == '/generate_mappings'}}
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Checkout PR
+      run: gh pr checkout $ISSUE
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        ISSUE: ${{ github.event.issue.html_url }}
+    - uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true
+    - name: Set up Podman
+      run: |
+        sudo apt-get update
+        sudo apt-get -y install podman
+    - name: Launch Qdrant server
+      run: |
+        port=6333
+        podman run -d -p ${port}:${port} qdrant/qdrant
+        echo "Waiting for Qdrant server to be ready..."
+        sleep 25
+    - uses: cue-lang/setup-cue@v1.0.0
+      with:
+        version: 'v0.7.0'
+    - name: Seed database
+      run: make seed
+    - name: Override Makefile with main branch version
+      run: |
+        git fetch origin main
+        git checkout origin/main -- Makefile
+        echo "Makefile has been overridden with the version from the main branch"
```

Comment:
Should we move this to the top? Basically enforce all make commands only run through main's version? Also lets us clump it next to the other git command which feels good for readability

Reply:
And when making that change, want to port it to the generate dist too?